### ISSUE TITLE
NMC Doesn't Remember/Find my username.

### ISSUE THREAD

**Issue by ianm32** [jaszhix/NoMansConnect#27]
I previously turned on "Protect Username."  Now upon starting, NMC displays a box to enter a username.  Upon entering anything and clicking "Save," it responds that the Protect Username feature must be turned off first.  The protection option is no longer present in the main menu, and my name is simply "Explorer."  

I have tried clearing the Roaming folder, to no avail.

**Comment by jaszhix** [comment]
For now, you can get around this bug by editing your username in settings.json in Roaming/NoMansConnect.

**Comment by jaszhix** [comment]
@ianm32 Can you give me more details? What did you do in the app before this occurred? How can it be reproduced?

**Comment by ianm32** [comment]
I wish I could.  I turned on username protection, and it worked as expected for a few days.  Then it just stopped and demonstrated the issue I described above.  I took your suggestion of editing the JSON and that indeed seems to have fixed it.  Even the username protection option has reappeared in the menu.

It's possible that my game crashed in such a way that something got messed up between NMC and my save file.  I'm using a lot of mods, so it does crash periodically.
